packages/genesys-spark-components/src/components/stable/gux-time-picker/gux-time-picker.tsx

```diff
@@ -78,6 +83,26 @@ export class GuxTimePicker {
   @Listen('blur')
   onBlur() {
     if (this.valueLastChange !== this.value) {
+      // Format input time to match format found in the popup time list (e.g. "01:30" -> "1:30", "00:30" -> "12:30", etc)
+      const split = this.value.split(':');
```

Comment:
This logic here needs some reworking. I'm wondering if we need to handle different time formats from the user or will the user have to input with a specific time format all the time? For instance could the user input "01:30" and also "13:30" and also "1:30"?